frontend/packages/dev-console/src/components/import/pipeline/PipelineTemplate.tsx

```diff
@@ -11,12 +11,17 @@ import { Pipeline } from '../../../utils/pipeline-augment';
 
 const MISSING_DOCKERFILE_LABEL_TEXT =
   'The pipeline template for Dockerfiles is not available at this time.';
-const MISSING_RUNTIME_LABEL_TEXT = 'There are no pipeline templates available for this runtime.';
 
 const labelType = 'pipeline.openshift.io/type';
 const labelRuntime = 'pipeline.openshift.io/runtime';
 const labelDocker = 'pipeline.openshift.io/strategy';
 
+const getAlertText = (isDockerStrategy, builderImage, resourceType) => {
+  if (isDockerStrategy) return MISSING_DOCKERFILE_LABEL_TEXT;
+
+  return `There are no pipeline templates available for ${builderImage} and ${resourceType} combination.`;
```

Comment:
This is not going to render a really understandable message... at least not one that is formatted to match the form.

I thought we had a map of ResourceType to renderable item, but it seems we copped out when we did the ResourceSection of the form (I was reviewing that, I should have requested it 😞). Oh wells, no sense in crying over spilled milk.

Need two changes:

* A map made next to the `Resources` enum that converts `resourceType` to a readable string
    * Go update `ResourceSection` to use that map (so we stay in sync)
* Thread `builderImages` into `PipelineSection` and down to here to allow you to get the `title` attribute

That should give you a readable string that makes sense with the rest of the form.

Reply:
Done.